resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -266,6 +266,14 @@ export default {
         }
     },
     watch: {
+        duration() {
+            this.setSpritesFolderpath();
+            this.updateThumbnailInterval();
+
+            this.spriteIdx = Math.floor(this.hoverTime / (this.thumbnailInterval * this.thumbnailsPerSprite));
```

Comment:
The spriteIdx is set directly in `updateSprite()`, so there is no need to set it here. It's the same in `mounted()` where the line can be removed.